command/crypto/nacl/box.go

```diff
@@ -43,6 +43,12 @@ receiver} sets are different. This is true even if the sets overlap. For
 example, a sender can use the same nonce for two different messages if the
 messages are sent to two different public keys.
 
+By default nonce are alphanumeric, but it's possible to use binary nonces using
```

Comment:
either "nonce is alphanumeric" or "nonces are alphanumeric" -- probably the latter